src/patient-registration/input/custom-input/openmrs-autosuggest/openmrs-autosuggest.css

```diff
@@ -0,0 +1,44 @@
+.suggestions {
```

Comment:
I'd suggest investigating if there's a carbon component that can be used for this purpose. Maybe this CSS could be reduced or eliminated.

Reply:
I did and didn't find one.

Reply:
@brandones so I had a meeting with @rbuisson today and together we tried to find an `auto-suggest` component but we couldn't find any. We tried to evaluate if there were any other components we could use for this purpose and we saw 
Search and ComboBox. Both seem to be inadequate and semantically wrong. For example the search box, doesn't show a list of suggestions. Looks more like a simple input box with some custom styling(the search icon and carbon look and feel). The ComboBox component seems to behave a little different from what we'll want (even though it might be possible to tweak it's behaviour) but the major block I noticed was that, the `ComboBox` just provides an `onChange` event handler. Now for an autosuggest there are two events happening and these events spit out different values. So for the first event which is the searching, the handler accepts the search string while when you click on a suggestion, the handler returns the selected item which would most likely be an object but the not sure the combobox can handle this. For reference of all this, below is a list of tickets that have requested for an autosuggest/autocomplete but were all closed without a solution and also a link to the documentation for the mentioned components. How do you suggest we proceed? cc @FlorianRappl 

https://github.com/carbon-design-system/carbon/issues/3513
https://github.com/carbon-design-system/carbon/issues/1605

https://www.carbondesignsystem.com/components/search/code/
https://react.carbondesignsystem.com/?path=/story/combobox--combobox

Reply:
@ivange94 Thanks to you and @rbuisson for such thorough investigation! Yes, I think it's good that we're creating our own autocomplete component. Thank you for your work on this.

I wonder if it should live in `esm-styleguide` or something like that? @FlorianRappl ? (either way it can be moved later)

Reply:
Fully agreed. `esm-styleguide` would be a great way to include it.